get-thin-edge_io.sh

```diff
@@ -121,7 +121,7 @@ minimal) install_basic_components ;;
 full)
     install_basic_components
     install_tedge_agent
-    if apt -v &>/dev/null; then
+    if apt -v >/dev/null 2>&1; then
```

Comment:
Are they the same?

Reply:
Yes, those are equal. See [here](https://www.shellcheck.net/wiki/SC3020)!